index.js

```diff
@@ -261,11 +262,10 @@ async function fastifyHttpProxy (fastify, opts) {
   }
 }
 
-fastifyHttpProxy[Symbol.for('plugin-meta')] = {
+module.exports = fp(fastifyHttpProxy, {
   fastify: '4.x',
-  name: '@fastify/http-proxy'
-}
-
-module.exports = fastifyHttpProxy
+  name: '@fastify/http-proxy',
+  encapsulate: true
```

Comment:
Omg, I didn't expect that but it makes sense

Good spot!

Reply:
Thanks 😊